docs/queries/byrole.mdx

```diff
@@ -164,6 +165,28 @@ the "👎" by calling `getByRole('link', { current: false })`. To learn more abo
 the current state see
 [ARIA `aria-current`](https://www.w3.org/TR/wai-aria-1.2/#aria-current).
 
+### `disabled`
+
+You can filter elements by their disabled state by setting `disabled: true` or
+`disabled: false`.
+
+For example in
+
+```html
+<body>
+  <section>
+    <button disabled>Login</button>
+    <div role="alert" aria-disabled="false">Failed to login</button>
```

Comment:
Having the `aria-disabled` here seems a bit confusing to me. 
Are we also filtering by `aria-disabled` or just by the `disabled` attribute? I think that we should explain it here so people will have a clear understanding.

Reply:
We're filtering by things that are disabled according to the a11y tree. This applies to every filter option in `ByRole`.

This here includes examples.

How would you explain it do avoid confusions?

Reply:
I'm thinking about something like this maybe?
https://github.com/testing-library/testing-library-docs/pull/1231/files#r1148318567